### PR TITLE
refactor(webdriverjs): rewrite webdriverjs to TS

### PR DESCRIPTION
**Pull request by michael-siek**
See: #195 

## Reviewer checks

**Required fields, to be filled out by PR reviewer(s)**

- [x] Code is reviewed for security

### CODE REVIEW

File: packages/webdriverjs/.eslintrc.js
Comment (stephenmathieson) [inline packages/webdriverjs/.eslintrc.js:3]: Why are you disabling this rule?

File: packages/webdriverjs/README.md
Comment (stephenmathieson) [inline packages/webdriverjs/README.md:128]: This reads as:

The class `AxeBuilder`'s method `analyze` returns a `Promise` which resolves to `axe.Results` or an `Error`, which is not correct. The `Promise` will never _resolve_ with an error, but it may `reject` instead.

IMO this should be:

```suggestion
### AxeBuilder#analyze(): Promise<axe.Results>
```

File: packages/webdriverjs/example.ts
Comment (stephenmathieson) [inline packages/webdriverjs/example.ts:8]: Thoughts on calling this file `example-with-callback.js` and creating an `example.js` which uses `async/await` to get results? At this point, I think most developers are familiar enough with `async/await` for the syntax to be used in the "main example".

File: packages/webdriverjs/package.json
Comment (stephenmathieson) [inline packages/webdriverjs/package.json:5]: I think it's fine to add yourself to `contributors`, but claiming ownership of this package probably won't fly...

File: packages/webdriverjs/package.json
Comment (stephenmathieson) [inline packages/webdriverjs/package.json:10]: Is this the only file we need in the tarball?

File: packages/cli/lib/axe-test-urls.js
Comment (stephenmathieson) [inline packages/cli/lib/axe-test-urls.js:4]: This indicates a breaking change. We can't ship this until we're ready for a SEMVER major release.

If we want this to land sooner than that, we'll want to do some hackery the TS:

```
export default AxeBuilder
```

should be:

```
export = AxeBuilder
```

Alternatly, we could add a `src/index.ts` which looks like:

```
import AxeBuilder from './AxeBuilder'

// Enable ESM/TS imports while we wait for a SEMVER major release.
AxeBuilder.default = AxeBuilder

export = AxeBuilder
```

**Review by stephenmathieson — CHANGES_REQUESTED** [review]
I don't think we can make breaking changes here right now.

File: packages/cli/lib/axe-test-urls.js
Comment (stephenmathieson) [inline packages/cli/lib/axe-test-urls.js:60]: This also indicates a breaking change. We can't do this either.

File: packages/webdriverjs/README.md
Comment (stephenmathieson) [inline packages/webdriverjs/README.md:50]: Why specify the encoding like this? Passing a string is _much_ more common here.

```suggestion
const axeSource = fs.readFileSync('./axe-1.0.js', 'utf8');
```

File: packages/webdriverjs/example-with-callback.ts
Comment (stephenmathieson) [inline packages/webdriverjs/example-with-callback.ts:1]: IMO we should leave the examples as regular/vanilla JavaScript. This allows them to be copy/pasted and executed without needing a build step.

File: packages/webdriverjs/package.json
Comment (stephenmathieson) [inline packages/webdriverjs/package.json:7]: Should leave the other contributors here.

File: packages/webdriverjs/src/index.ts
Comment (stephenmathieson) [inline packages/webdriverjs/src/index.ts:8]: Why are you exporting this here and again at the end of the file?

**Review by stephenmathieson — COMMENTED** [review]
Small stuff. Looking good tho 👍

**Review by stephenmathieson — COMMENTED** [review]
LGTM 👍 

You should get an OK from @jeeyyy, @WilcoFiers or @AdnoC before merging tho.

**Review by jeeyyy — APPROVED** [review]

**Pull request merged**
